doc/release/RELEASE-NOTES.md

```diff
@@ -17,7 +17,11 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe scheme changes here.*
 
 ### Code/API changes
-* *Describe code/API changes here.*
+* [OSDEV-994](https://opensupplyhub.atlassian.net/browse/OSDEV-994) API. Update to pass all merge events to user based on contrib id. A non-admin API user makes:
+- a GET call to /moderation-events/merge/
+and receives information about merges that have occurred for all contributors.
+- a GET call to /moderation-events/merge/?contributors=1078&contributors=20&contributors=2608
```

Comment:
It's better to keep default placeholders here `contributors=<id_number_x>&contributors=<id_number_y>&contributors=<id_number_y>`

Reply:
replaced